packages/next/server/config-shared.ts

```diff
@@ -507,6 +504,9 @@ export interface NextConfig extends Record<string, any> {
 
   output?: 'standalone'
 
+  // A list of packages that should always be transpiled and bundled in the server
+  transpilePackages?: string[]
```

Comment:
Do we want to warn when the experimental config is used and automatically map it simiilar to these:

https://github.com/vercel/next.js/blob/2b3a38ee472e6d0854c35cc46f12f8bad769d654/packages/next/server/config.ts#L545-L551

Reply:
Yep!

Reply:
Updated!